### DISCUSSION TITLE
Android 13 compatibility

### DISCUSSION THREAD

**Discussion by ohlive** [capacitor-community/bluetooth-le#453] [General]
Hi, 
On different smartphone like samsung, its up on android13.

I upgrade target sdk version on variables.gradle to 
 minSdkVersion = 22
    compileSdkVersion = 33
    targetSdkVersion = 33

Dont work .

Someone have the same problems?

**Comment by pwespi** [comment]
Generally it is recommended to use the variables of the latest Capacitor release: https://github.com/ionic-team/capacitor/blob/main/android-template/variables.gradle.

What exactly doesn't work on Android 13?

**Reply by ohlive** [reply]
Thx for your reply.

I forgot to up sdk target on Manifest.xml

`<uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" android:maxSdkVersion="33" />`
`<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" android:maxSdkVersion="33" />`

Now is ok with that.
